content/22.vaccines.md

```diff
@@ -316,7 +315,37 @@ In a phase III randomized, observer-blinded, placebo-controlled clinical trial i
 Novavax has since been approved in several places, including the United Kingdom [@url:https://www.gov.uk/government/news/novavax-covid-19-vaccine-nuvaxovid-approved-by-mhra] and the E.U. [@url:https://www.ema.europa.eu/en/news/ema-recommends-nuvaxovid-authorisation-eu] and applied for an EUA from the FDA in early 2022 [@url:https://ir.novavax.com/2022-01-31-Novavax-Submits-Request-to-the-U-S-FDA-for-Emergency-Use-Authorization-of-COVID-19-Vaccine]. 
 Although Novavax promised to provide up to 1.1 billion doses to COVAX; however, there have been reports of struggle to maintain production capacity and quality, hindering its production targets [@url:https://www.politico.com/news/2021/10/19/novavax-vaccine-rush-process-global-campaign-516298].
 
-<!--Add summary of Covifenz https://covid19.trackvaccines.org/vaccines/26/ as an example of VLP-->
+The leading example of a VLP approach applied to COVID-19 comes from Covifenz, a VLP vaccine developed by Canadian company Medicago [@doi:10.1503/cmaj.1095992].
+This vaccine was developed using plant-based VLP technology [@url:https://medicago.com/en/our-products/our-vaccines/covifenz-covid-19-vlp-vaccine] that the company had been investigating in order to develop a high-throughput quadrivalent VLP platform to provide protection against influenza [@doi:10/gjn28x].
+The approach utilizes _Nicotiana benthamiana_, an Australian relative of the tobacco plant, as an upstream bioreactor [@doi:10/gjn28x; @doi:10.1038/s41591-021-01370-1; @doi:10.1038/s41591-021-01370-1].
+Specifically, the _S_ gene from SARS-CoV-2 in its prefusion conformation is inserted into a bacterial vector (_Agrobacterium tumefaciens_) that then infects the tobacco plant cells [@doi:10/gjn28x; @doi:10.1038/s41591-021-01370-1].
```

Comment:
Do you still refer to it as tobacco if it is only a relation to tabocco?

Reply:
You're right, it's called "benthi"!
```suggestion
Specifically, the _S_ gene from SARS-CoV-2 in its prefusion conformation is inserted into a bacterial vector (_Agrobacterium tumefaciens_) that then infects the plant cells [@doi:10/gjn28x; @doi:10.1038/s41591-021-01370-1].
```